cmd/convert.go

```diff
@@ -36,7 +36,8 @@ Note:
   resources.
 
 Example:
-  terraform-validator convert ./example/terraform.tfplan --project my-project
+  terraform-validator convert ./example/terraform.tfplan --project my-project \
+    --ancestry organization/my-org/folder/my-folder/project/my-project
```

Comment:
@yukinying Specifying the project in both `--project` and `--ancestry` seems redundant. Ancestry should just be the parent path (`organization/my-org/folder/my-folder`).

Reply:
Agree. This is redundant. Code updated to add project to ancestry.